test/functional/apps/context/_filters.js

```diff
@@ -43,18 +43,18 @@ export default function ({ getService, getPageObjects }) {
     it('should be addable via expanded doc table rows', async function () {
       await docTable.toggleRowExpanded({ isAnchorRow: true });
 
-      const anchorDetailsRow = await docTable.getAnchorDetailsRow();
-      await docTable.addInclusiveFilter(anchorDetailsRow, TEST_ANCHOR_FILTER_FIELD);
-      await PageObjects.context.waitUntilContextLoadingHasFinished();
-
-      await docTable.toggleRowExpanded({ isAnchorRow: true });
-
       await retry.try(async () => {
-        expect(await filterBar.hasFilter(TEST_ANCHOR_FILTER_FIELD, TEST_ANCHOR_FILTER_VALUE, true)).to.be(true);
+        const anchorDetailsRow = await docTable.getAnchorDetailsRow();
+        await docTable.addInclusiveFilter(anchorDetailsRow, TEST_ANCHOR_FILTER_FIELD);
+        await PageObjects.context.waitUntilContextLoadingHasFinished();
+        // await docTable.toggleRowExpanded({ isAnchorRow: true });
+        expect(
+          await filterBar.hasFilter(TEST_ANCHOR_FILTER_FIELD, TEST_ANCHOR_FILTER_VALUE, true)
+        ).to.be(true);
```

Comment:
Apart from auto formatting, the main change here is to move the command of expanding a row into the retry section, because of async rendering of the table tab (Angular). this async rendering will soon be replaced by React in this case, but anyway it's better and more stable to implement it this way